Makefile

```diff
@@ -57,6 +61,23 @@ go-get:
 	go get -u github.com/nfnt/resize
 	go get -u github.com/coreos/bbolt
 
+.PHONY: go-get-windows
+go-get-windows:
+	go get -u golang.org/x/sys/unix
+	go get -u gopkg.in/yaml.v2
+	go get -u github.com/kidoman/embd
+	go get -u github.com/shirou/gopsutil
+	go get -u github.com/gorilla/mux
+	go get -u github.com/gorilla/sessions
+	go get -u gopkg.in/robfig/cron.v2
+	go get -u github.com/dustin/go-humanize
+	go get -u github.com/reef-pi/rpi/pwm
+	go get -u github.com/reef-pi/drivers
+	go get -u github.com/reef-pi/adafruitio
+	go get -u github.com/nfnt/resize
+	go get -u github.com/coreos/bbolt
+	go get -u github.com/StackExchange/wmi
+
```

Comment:
@Ranthalion I think we can keep the common bits in a common make target, and have the windows task depend on that, and explicitly calls out only go get stackexchange/wmi , that will remove some duplication. What do you think?